Blimp/Blimp.h

```diff
@@ -190,7 +190,7 @@ class Blimp : public AP_Vehicle
             uint8_t initialised_params      : 1; // 24      // true when the all parameters have been initialised. we cannot send parameters to the GCS until this is done
             uint8_t unused3                 : 1; // 25      // was compass_init_location; true when the compass's initial location has been set
             uint8_t unused2                 : 1; // 26      // aux switch rc_override is allowed
-            uint8_t armed_with_switch       : 1; // 27      // we armed using a arming switch
```

Comment:
you could probably just zapped all these unused items because they've just been inherited from Copter.

Reply:
True!

But focus required :-)